### ISSUE TITLE
Cannot read property 'getPanelInfoById' of undefined

### ISSUE THREAD

**Issue by yosida95** [cloudflare/cloudflare-grafana-app#10]
After upgrading Grafana to v6.0.0 (commit: 34a9a62), all DNS Firewall panels stop working with an error `Cannot read property 'getPanelInfoById' of undefined`. Installed cloudflare-grafana-app version is v0.1.3.

```
metrics_panel_ctrl.ts:178 Panel data error: TypeError: Cannot read property 'getPanelInfoById' of undefined
    at CloudflareDatasource.query (datasource.js:61)
    at t.issueQueries (metrics_panel_ctrl.ts:234)
    at module.ts:24
    at angular.js:17051
    at angular.js:17095
    at l.$digest (angular.js:18232)
    at angular.js:18462
    at h (angular.js:6362)
    at angular.js:6642
```

**Comment by daenney** [comment]
This has already been fixed in #9. Unfortunately a new release hasn't been published yet.

**Comment by Ryonez** [comment]
I have this same issue.

**Comment by daenney** [comment]
This is fixed now with #11.

**Comment by yosida95** [comment]
I have confirmed that this issue was fixed in v0.1.4. I will close this issue. Thanks all.
